src/share/classes/jdk/jfr/internal/Options.java

```diff
@@ -49,6 +49,8 @@
     private static final boolean DEFAULT_SAMPLE_THREADS = true;
     private static final long DEFAULT_MAX_CHUNK_SIZE = 12 * 1024 * 1024;
     private static final SafePath DEFAULT_DUMP_PATH = SecuritySupport.USER_HOME;
+    private static final boolean DEFAULT_SAMPLE_OBJECT_ALLOCATIONS = false;
+    private static final long DEFAULT_OBJECT_ALLOCATIONS_SAMPLING_INTERVAL = 1024;
```

Comment:
this magic number has been defined twice, in vm and in class library, can we just do that once? which may help to prevent future chances of inconsistency

Reply:
other fields are also defined twice, such as DEFAULT_MAX_CHUNK_SIZE.